server/api/operator_test.go

```diff
@@ -147,6 +147,11 @@ func mustPutStore(c *C, svr *server.Server, id uint64, state metapb.StoreState,
 		},
 	})
 	c.Assert(err, IsNil)
+	_, err = svr.StoreHeartbeat(context.Background(), &pdpb.StoreHeartbeatRequest{
+		Header: &pdpb.RequestHeader{ClusterId: svr.ClusterID()},
+		Stats:  &pdpb.StoreStats{StoreId: id},
+	})
+	c.Assert(err, IsNil)
```

Comment:
Why do we need these `svr.StoreHeartbeat`?

Reply:
The Builder considers store status (disconnected, busy, etc.) to determine if it can accept new leaders (previously only check reject leader property). Without hearbeat, the store is considered disconnected and fail some tests.